test/config/checkov.yaml

```diff
@@ -10,6 +10,8 @@ framework:
 skip-download: false
 output: cli
 quiet: true
-skip-check: []
+skip-check:
```

Comment:
I suggest to submit a PR to deactivate those checks. I would also add the check that are currently throwing an error, but put a comment before to mention that those should be fixed. The same PR would enable checkov in the CI test.

That would prevent us from introducing new errors while the existing one are solved.